packages/malloy/src/lang/test/parse.spec.ts

```diff
@@ -82,7 +82,7 @@ function badModel(s: MarkedSource | string, msg: string): TestFunc {
     if (t.translated) {
       fail(emsg);
     } else {
-      const errList = m.errors().errors;
+      const errList = m.problems().problems;
```

Comment:
problems() should return the list and problemResponse() should be protected and return the response.